Source/MonoGame.Extended/Collections/Poolable.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MonoGame.Extended.Collections
+{
+    public abstract class Poolable : IPoolable
```

Comment:
Ugh, to do that I need the `IPoolable` to have a reference to the `ObjectPool` but `ObjectPool` is a generic... Screw it, the API is ok as is with the `ObjectPool` to return the `IPoolable`.